mne/connectivity/envelope.py

```diff
@@ -51,7 +55,8 @@ def envelope_correlation(data, combine='median', verbose=None):
     from scipy.signal import hilbert
     corrs = list()
     n_nodes = None
-    _validate_type(combine, (str, None), 'combine')
+    if not callable(combine):
+        _validate_type(combine, (str, None), 'combine')
```

Comment:
This should be refactored with the code in `evoked` (or wherever we recently added `trim_mean` support, etc.) with a private function / the same set of options.